main.go

```diff
@@ -16,7 +16,7 @@ func init() {
 	log.SetOutput(os.Stderr)
 
 	// Only log the error severity or above.
-	log.SetLevel(log.ErrorLevel)
+	log.SetLevel(log.DebugLevel)
```

Comment:
will buildkite only pick up the stdout - have we tested that?

Reply:
Except for this, all other comments have been addressed; Testing is in progress once completed I will notify you. Thanks @prezha

Reply:
pipeline is picking both ```stdout``` and ```stderr``` and hence we may need to set the log level to ErrorLevel....This change will update it in the other PR that is awaiting acceptance.(#pr25)
[bk-log](https://buildkite.com/equinix/buildkite-actions-sandbox/builds/73)
[bk-log1](https://buildkite.com/equinix/buildkite-actions-sandbox/builds/74)